forte/utils/utils_io.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2019 The Forte Authors. All Rights Reserved.
```

Comment:
I like this `utils_io.py` file here. However, there is another similarly named `io_utils.py` in `forte/data`. I am wondering if we can consolidate the IO related functions at one place.

Reply:
Delete `ensure_dir` and rename `io_utils` to `data_utils_io`